lib/drives/pages/drive_chat_page.dart

```diff
@@ -18,130 +18,101 @@ class DriveChatPage extends StatefulWidget {
 
 class _DriveChatPageState extends State<DriveChatPage> {
   Drive? _drive;
-  bool _fullyLoaded = false;
 
   @override
   void initState() {
     super.initState();
     setState(() {
       _drive = widget.drive;
     });
-    loadDrive();
   }
 
   @override
   Widget build(BuildContext context) {
     List<Widget> widgets = [];
-    if (_fullyLoaded) {
-      Set<Profile> riders = _drive!.approvedRides!.map((ride) => ride.rider!).toSet();
-      List<Ride> pendingRides = _drive!.pendingRides!.toList();
-      if (riders.isEmpty && pendingRides.isEmpty) {
-        return Scaffold(
-          appBar: AppBar(
-            title: Text(S.of(context).pageDriveChatTitle),
-          ),
-          body: Center(
-            child: Text(S.of(context).pageDriveChatEmptyMessage),
-          ),
-        );
-      } else {
-        if (riders.isNotEmpty) {
-          List<Widget> riderColumn = [
-            const SizedBox(height: 5.0),
-            Text(
-              S.of(context).pageDriveChatRiderHeadline,
-              style: Theme.of(context).textTheme.titleLarge,
-            ),
-            const SizedBox(height: 10.0),
-            _riderList(riders),
-          ];
-          widgets.addAll(riderColumn);
-        }
-        widgets.add(const SizedBox(height: 10.0));
-        if (pendingRides.isNotEmpty) {
-          List<Widget> pendingRidesColumn = [
-            const SizedBox(height: 5.0),
-            Text(S.of(context).pageDriveChatRequestsHeadline, style: Theme.of(context).textTheme.titleLarge),
-            const SizedBox(height: 10.0),
-          ];
-          pendingRidesColumn.addAll(_pendingRidesList(pendingRides));
-          widgets.addAll(pendingRidesColumn);
-        }
-      }
+    Set<Ride> approvedRides = _drive!.approvedRides!.toSet();
+    List<Ride> pendingRides = _drive!.pendingRides!.toList();
```

Comment:
I'm confused, why is this dependent on pending rides?

Reply:
sorry forgot that. But that should actually be part of refactor-pending-rides i guess